queue_job_chunk/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Creu Blanca
```

Comment:
Wrong copyright.
Use snippet instead of copy paste, it's faster and avoid error